README.md

```diff
@@ -42,10 +42,12 @@ This application will be responsible for creating a User Interface that will all
 # Running the entire application
 
 1. Clone down the repository
-2. Run sql code to create the required tables
+2. Run sql code to create the required tables 
 3. Open the project and __navigate into the `front-end` directory__, then run `npm install`
 4. After it has finished installing, __navigate back to the root directory__, and run `mvn clean install`
 5. Once the project has finished building, then you can run the executable jar file. (Below)
 (Depending on terminal '\' might need to be changed to '/') 
 `java -jar .\target\rules-engine-0.0.1-SNAPSHOT.jar`  or `java -jar ./target/rules-engine-0.0.1-SNAPSHOT.jar`
-6. Go to the following URl 'http://localhost:8080/#/create-ruleset'
\ No newline at end of file
+6.
+   - Go to the following URl 'http://localhost:8080/#/create-ruleset' to create a rule
+   - Go to the following URl 'http://localhost:8080/#/update-ruleset/{insert id of ruleset here}' to edit a rule
```

Comment:
Having the "insert id of ruleset here" inside the quotes and the braces makes it display weirdly in the read version. 
I would rewrite this as:
"Go to the following URl 'http://localhost:8080/#/update-ruleset/{id}' to edit a rule. (Id should be replaced with the id of the rule you want to edit)"